digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/ViiteApi.scala

```diff
@@ -1008,7 +1010,8 @@ class ViiteApi(val roadLinkService: RoadLinkService, val vVHClient: VVHClient,
             "status" -> splittedLinks.status.value,
             "roadTypeId" -> splittedLinks.roadType.value,
             "discontinuity" -> splittedLinks.discontinuity.value,
-            "elyCode" -> splittedLinks.ely
+            "elyCode" -> splittedLinks.ely,
+            "roadName" -> splittedLinks.roadName.getOrElse("")
```

Comment:
The value here I believe it shouldn't be an Option[String] anymore. If we handle everything on service level should be better

Reply:
Would require changing the whole ProjectLink object in order to do it, making it a permanent String object then it would be detrimental in the cases that the project links has no name.